pyupgrade.py

```diff
@@ -1537,6 +1535,37 @@ def generic_visit(self, node):  # type: (ast.AST) -> None
         super(FindPy3Plus, self).generic_visit(node)
 
 
+def reversed_walk(node):  # type: (ast.AST) -> Generator[ast.AST, None, None]
+    todo = [node]
+    while todo:
```

Comment:
@asottile would it be okay to add `pragma: no branch` (for it's `while True`) here and further walk-branches? Or it should be refactored?

Reply:
you should be able to hit the coverage here -- the missing branch indicates this either never terminates or never loops more than once

Reply:
Addressed in 81dc036f6a52e32a3dd1e992334c24fc09cea12b.